lib/sarif/base_sarif.rb

```diff
@@ -30,7 +30,10 @@ def build_tool(rules: [])
           "name" => @scan_report.scanner_name,
           "version" => @scan_report.version,
           "informationUri" => @uri,
-          "rules" => rules
+          "rules" => rules,
+          "properties" => {
+            "salusEnforced": @required || false
```

Comment:
IIRC the || false was to ensure we had proper support here for unsupported (scanners that don't yet have a sarif adapter present) scanners.  If that's the case can you include one spec for that case?  Beyond that looks great!

Reply:
Yep that's the case, actually I just remembered, the unsupported scanner case is already being tested in base_sarif_spec.rb around lines 12-20. I'd be happy to add another spec if this is insufficient!